server/src/test/java/ServerTest.java

```diff
@@ -42,8 +43,9 @@ public static void main(String[] args) {
         RpcServer rpcServer = new RpcServer(workingThreads);
         rpcServer.setHandler(new DefaultCoordinator(rpcServer));
```

Comment:
please change this class to a test case, not main method.

Reply:
Hi, according your advice, I have done with it.  but in server module , the original authors's tests used the Main method, you can view the original code for this , so I want to know that we need to be consistent with him, or should just fix the bug. THX.

Reply:
> please change this class to a test case, not main method.

for this situation,i thinks it's not necessary to use @Test.otherwise,the server will never be closed.

Reply:
you can use @ after to shutdown the test server

Reply:
after the server started,the main thread will be block, the @after will not get a chance to execute.

Reply:
@xingfudeshi Then should  I change the @Test method back to Main method which the original author does, and we just focus on the bugfix?

Reply:
yes.

Reply:
@xingfudeshi It had already done.

Reply:
@leizhiyuan Hi, Add @Test and @After method seem not suitable for this case, No as good as we focus on bugfix. So I change back into Main mothod as the original author does.

Reply:
@leizhiyuan Hi, since we have common in changes after we discussed before, please approve these changes, not request changes. after this, changes can be merge into develop branch, I think. Thanks all